portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -1760,242 +2063,286 @@ definition {
+			DMNavigator.gotoDocumentCP(dmDocumentTitle = "DM Document Title");
 
-		DMDocument.expandInfo();
+			DMDocument.expandInfo();
 
-		var dmDocumentURL = selenium.getAttribute("//input[contains(@id,'urlInput')]@value");
+			var dmDocumentURL = selenium.getAttribute("//input[contains(@id,'urlInput')]@value");
 
-		var editDocumentURL = StringUtil.extractFirst("${dmDocumentURL}", "jpg");
+			var editDocumentURL = StringUtil.extractFirst("${dmDocumentURL}", "jpg");
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+		task ("And: Create a new WC with the saved URL as reference in the source content") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
 
-		WebContentNavigator.gotoAddCP();
+			WebContentNavigator.gotoAddCP();
 
-		PortletEntry.inputTitle(title = "Web Content Title");
+			PortletEntry.inputTitle(title = "Web Content Title");
 
-		CKEditor.addSourceContent(content = '''<a href="${editDocumentURL}jpg">Link to Document</a>''');
+			CKEditor.addSourceContent(content = '''<a href="${editDocumentURL}jpg">Link to Document</a>''');
 
-		PortletEntry.publish();
+			PortletEntry.publish();
+		}
 
-		ProductMenu.gotoPortlet(
-			category = "Publishing",
-			portlet = "Export");
+		task ("When: User export with default settings") {
+			ProductMenu.gotoPortlet(
+				category = "Publishing",
+				portlet = "Export");
 
-		LexiconEntry.gotoAdd();
+			LexiconEntry.gotoAdd();
 
-		LAR.exportWithAssertionOnSuccess();
+			LAR.exportWithAssertionOnSuccess();
 
-		var larFileName = LAR.getLarFileName();
+			var larFileName = LAR.getLarFileName();
 
-		LAR.downloadLar();
+			LAR.downloadLar();
+		}
 
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("And: Import the LAR file on a freshly created site") {
+			JSONGroup.addGroup(groupName = "Site Name");
 
-		LAR.importSiteCP(
-			larFileName = "${larFileName}",
-			siteName = "Site Name");
+			LAR.importSiteCP(
+				larFileName = "${larFileName}",
+				siteName = "Site Name");
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name");
+		task ("Then: The imported WC should have the same source content as when it was created, without the UUID in the pasted URL") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-name");
 
-		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+			WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
 
-		CKEditor.assertSourceContent();
+			CKEditor.assertSourceContent();
+		}
 	}
 
 	@description = "This is a use case for LPS-84290. Verify is possible export Home Folder Permissions."
 	@priority = "3"
 	test ImportSiteHomeFolderPermissions {
-		JSONGroup.addGroup(groupName = "Site Name");
+		task ("Given: User adds a new site") {
+			JSONGroup.addGroup(groupName = "Site Name");
+		}
 
-		Navigator.openURL();
+		task ("When: User removes documents and media permissions from Site member on default site") {
```

Comment:
Make it more explicit, e.g. "When: Remove the Home Folder Permissions from DM portlet for site member role on default site"